duckduckgo/iOS
# allow delete from edit screen

Task/Issue URL: https://app.asana.com/0/72649045549333/1205194478809824/f
Tech Design URL:
CC:

**Description**:

Adds:
* Delete button to edit bookmark screen
* Enable/disable Save button on add/edit login screen
* Updated trash icon

**Steps to test this PR**:

Edit Bookmark:
1. Add some bookmarks.
2. Edit a bookmark. 
3. Tap delete
4. Click OK
5. Check that bookmark is deleted
6. Create a favorite
7. Repeat above test, check favorite is deleted
8. Repeat above test with favorite and tap undo on the action message.  Check that bookmark/favorite is returned to correct folder / favorite position. 
9. Repeat undo test but close the bookmarks view controller before tapping undo. 

Add/Edit Login:
1. Manually add a new login, save button should be visible but disabled
2. Add some text to at least one field (try each field) and the save button should enable
3. Edit an existing login, same logic should apply - only when at least one field has text should the save button be enabled

Trash Icon:
1. Go to bookmarks list
2. Swipe left on a bookmark to show delete icon
3. Should be updated Trash icon





**Copy Testing**:


**Orientation Testing**:


**Device Testing**:


**OS Testing**:


**Theme Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
